### PR TITLE
Travis to Xcode 10.2

### PR DESCRIPTION
**Pull request by paulb777**

### CODE REVIEW

File: scripts/if_cron.sh [hunk 1]
```diff
@@ -18,7 +18,7 @@
 #   - TRAVIS_EVENT_TYPE - to check if this is a cron job
 #
 
-if [[ "$TRAVIS_EVENT_TYPE" == "cron" ]]; then
+if [[ "$TRAVIS_EVENT_TYPE" != "cron" ]]; then
```
Comment (ryanwilson) [inline scripts/if_cron.sh]: Was this intentional?

Comment (paulb777) [reply scripts/if_cron.sh]: Yes temporarily, I wanted to test everything.

**Comment by paulb777** [comment]
There seems to be an issue with xcodebuild and Xcode 10.2 and our project where macOS and tvOS try to build as part of iOS targets. It works fine from the IDE. I'll put this PR on hold for now.

See https://travis-ci.org/firebase/firebase-ios-sdk/builds/516665084?utm_source=github_status&utm_medium=notification

**Comment by paulb777** [comment]
More about the Xcode 10.2 issue and CocoaPods at https://github.com/CocoaPods/CocoaPods/issues/8729

**Comment by paulb777** [comment]
Closed in favor of #3714

**Pull request closed without merging**